blackbox/docs/appendices/release-notes/3.1.2.rst

```diff
@@ -0,0 +1,67 @@
+.. _version_3.1.2:
+
+=============
+Version 3.1.2
+=============
+
+Released on 2018/10/18.
+
+.. NOTE::
+
+    If you are upgrading a cluster, you must be running CrateDB 2.0.4 or higher
+    before you upgrade to 3.1.2.
+
+    We recommend that you upgrade to the latest 3.0 release before moving to
+    3.1.2.
+
+    You cannot perform a `rolling upgrade`_ to this version. Any upgrade to this
```

Comment:
A rolling upgrade from `3.1.1` to `3.1.2` is supported

Reply:
3.1.0 was a testing release and never marked as stable. That's why the information there is correct as we don't support rolling upgrade from testing release to stable release.
3.1.1 is stable, so a rolling upgrade from it should be possible.

Reply:
Ah, that makes sense. Thank you. :slightly_smiling_face: